### ISSUE TITLE
If no JavaScript is available Gutenberg is only showing an admin notice that Javascript is required instead of the actual content

### ISSUE THREAD

**Issue by rpkoller** [WordPress/gutenberg#12844]
I've already pointed out the details in https://github.com/WordPress/gutenberg/issues/2719 over a year ago that the editor should provide a progressive enhancement experience. Meaning never ever should the content be inaccessible by the user. There are many scenarios that Javascript might not be available https://kryogenix.org/code/browser/everyonehasjs.html not only if the functionality is actually switched off in the browser. The issue got agreed on by several stakeholders over the course of several months but in the end got hi-jacked by the half baked makeshift admin notices solution which isnt fixing the initial ticket at all. @mkaz recommended that i shall open a new ticket again.

**Comment by karmatosed** [comment]
I am removing the 'User Experience (UX)' label as part of a label cleanup. It's not being used anymore consistently so let's try and keep to 'needs design' and 'needs design feedback'. If we find a need for another label we can consider it but having those 2 should cover this.

**Comment by sarahmonster** [comment]
The design team discussed this during a [triage session in Slack today](https://wordpress.slack.com/archives/C02S78ZAL/p1559059988063400) (Note: You may need a [Slack account](https://make.wordpress.org/chat/) to log in.)

Overall consensus was that this would be great from a design perspective, but there may be architectural hurdles to accomplishing this well. We'd love some feedback from a technical perspective here.

**Comment by rpkoller** [comment]
@sarahmonster the technical feedback was already given back in september 2017 by @ellatrix in the original issue: https://github.com/WordPress/gutenberg/issues/2719#issuecomment-331896425

**Comment by youknowriad** [comment]
Closing as not-planned: the technical assessment from the very beginning stands — the block editor is a JavaScript application and a no-JS read-only rendering isn't a direction the project pursued, with no movement since the 2019 design triage. Sites needing no-JS editing paths have the classic editor plugin. Thanks for advocating for resilience!
